com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity

```diff
@@ -0,0 +1,758 @@
+%YAML 1.1
```

Comment:
This should be in the game view on a canvas

![image](https://user-images.githubusercontent.com/6424658/214966851-9a06b33c-6fea-448c-a957-a79cebc47428.png)